src/components/Dictionary/DictionaryPresentation.js

```diff
@@ -30,6 +30,7 @@ function DictionaryPresentation({
     link: 'flex items-center transition duration-500 ease-in-out p-2 grow rounded-lg capitalize cursor-pointer text-lg xl:text-xl text-fv-charcoal',
     icon: 'inline-flex fill-current w-6 xl:w-8 mr-2 xl:mr-5',
   }
+  const count = items?.pages[0]?.count
```

Comment:
Optional: count variable can be extracted in the Data layer and then passed through instead of being extracted here.